semcore/wizard/src/Wizard.jsx

```diff
@@ -164,8 +179,14 @@ function Stepper(props) {
         e.preventDefault();
         onActive(step, e);
       }
+      if (e.key === 'ArrowUp') {
```

Comment:
arrow navigation works well!
but we should remove navigation through the tablist with the Tab key, Tab should move focus out of the tablist